.github/workflows/e2eNew.yml

```diff
@@ -0,0 +1,236 @@
+name: E2E Test Suite - ESM
+
+on:
+  workflow_run:
+    workflows:
+      - Nightly Build Develop
+      - Test, Build, and Release
+    types:
+      - completed
+
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'testesm'
+        type: string
+      anInitialSuite:
+        description: 'Verify Extensions'
+        required: false
+        default: true
+        type: boolean
+      sObjectsDefinitions:
+        description: 'sObjects Definitions'
+        required: false
+        default: true
+        type: boolean
+      templates:
+        description: 'Create Commands'
+        required: false
+        default: true
+        type: boolean
+      manifestBuilder:
+        description: 'Manifest Builder'
+        required: false
+        default: true
+        type: boolean
+      pushAndPull:
+        description: 'Push and Pull'
+        required: false
+        default: true
+        type: boolean
+      auraLsp:
+        description: 'Aura LSP'
+        required: false
+        default: true
+        type: boolean
+      vscodeVersion:
+        description: 'VSCode Version'
+        required: false
+        default: '1.90.2'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: true
+        type: string
+
+  workflow_call:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'testesm'
+        type: string
+      anInitialSuite:
+        description: 'Verify Extensions'
+        required: false
+        default: true
+        type: boolean
+      sObjectsDefinitions:
```

Comment:
@CristiCanizales shouldn't the defaults be false for test jobs that have not been finished?

Reply:
I didn't get this... finished how?